timesketch.conf

```diff
@@ -61,3 +61,7 @@ UPLOAD_FOLDER = u'/tmp'
 # server is running.
 CELERY_BROKER_URL='redis://ip:port',
 CELERY_RESULT_BACKEND='redis://ip:port'
+
+# Path to Plaso WinEvtx message string database.
```

Comment:
I think its better to refer to this as just the "plaso data directory" - there could be more things added in the future.